components/ntp_background_images/browser/ntp_background_images_data.h

```diff
@@ -12,10 +12,13 @@
 #include "base/files/file_path.h"
 #include "base/values.h"
 #include "third_party/abseil-cpp/absl/types/optional.h"
-#include "third_party/skia/include/core/SkColor.h"
 #include "ui/gfx/geometry/point.h"
 #include "ui/gfx/geometry/rect.h"
 
+#if !BUILDFLAG(IS_IOS)
+#include "third_party/skia/include/core/SkColor.h"
+#endif
+
```

Comment:
optional - I realized that this header including is not necessary for desktop also. If you want, you can just delete this.

Reply:
Sounds good, will remove when rebasing after https://github.com/brave/brave-core/pull/18706 merges as that will also include a few changes :)